### DISCUSSION TITLE
🙂 Thank you for letting me help you here!

### DISCUSSION THREAD

**Discussion by c1972** [thorsten/phpMyFAQ#3036] [General]
I really enjoy testing and describing errors. Making suggestions from time to time (even if they sometimes aren't very useful 🤪). And for a useful project like this... great! 😎

Regards Christian

Mal bisschen Deutsch wieder:
- Ursprünglich hatte ich mir im webgo angesehn, was für Software es da noch gibt. Da ist unter Anderem **phpMyFAQ** mit dabei.
- Ich hatte mehrfach **vergeblich** versucht, den 1-Click-Installer dafür zu benutzen. Dann hatte ich den webgo-Support gefragt und die haben auch keine Lösung dafür gefunden. Also hab ich's mir dann selbst installiert (was wohl ohnehin mehr Sinn macht). 
- Ich hatte übrigens den Eindruck, dass zumindest dieser eine webgo-Mitarbeiter nicht sonderlich an einer Lösung interessiert war. Obwohl ich mit dem webgo-Support ansonsten sehr zufrieden bin.

**Comment by thorsten** [comment]
It's always welcome. If you like, you can join @modelrailroader and me at https://discord.gg/wszhTceuNM
